backend/app/routers/chat.py

```
@@ -346,30 +346,59 @@ def _maybe_create_alert_from_nl(nl: str, parsed: Dict[str, Any]) -> Optional[Dic
     obj = parsed.get("objects.object_name") or "person"
     color = parsed.get("objects.color")
     cam = parsed.get("camera_id")
-    count = _guess_count_from_text(nl) or 1
+    
+    # Use count_constraint from parsed if available, otherwise guess
+    count_constraint = parsed.get("count_constraint")
+    count_dict = {}
+    if count_constraint:
+        for k, v in count_constraint.items():
+            if k == "eq": count_dict["=="] = v
```

Ruff (0.15.1)

[error] Multiple statements on one line (colon)

(E701)